MapboxSearch/sample/src/main/java/com/mapbox/search/sample/api/AddressAutofillKotlinExampleActivity.kt

```diff
@@ -16,9 +15,7 @@ class AddressAutofillKotlinExampleActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        addressAutofill = AddressAutofill.create(
```

Comment:
Do all of the examples work for you? These examples are part of public documentation, this would be great to tell the customers how to set token.